fab/pillows/swiss.yml

```diff
@@ -5,3 +5,5 @@ exclude_pillows:
   - ReportCaseToElasticsearchPillow
   - ReportXFormToElasticsearchPillow
   - CaseSearchToElasticsearchPillow
+  - kafka-ucr-static
+  - kafka-ucr-main
```

Comment:
yeah was wondering how sneaky this would be. one thought would be to have
some sort of error when trying to create ucr's on swiss until we remove
this (maybe a soft assert?)

On Wed, Sep 14, 2016 at 12:30 AM, Cory Zue <notifications@github.com> wrote:

> In fab/pillows/swiss.yml
> <https://github.com/dimagi/commcare-hq-deploy/pull/115#discussion_r78684818>
> :
>
> > @@ -5,3 +5,5 @@ exclude_pillows:
> >    - ReportCaseToElasticsearchPillow
> >    - ReportXFormToElasticsearchPillow
> >    - CaseSearchToElasticsearchPillow
> > +  - kafka-ucr-static
> > +  - kafka-ucr-main
>
> sounds good. also sounds fine to remove the UCR queues for now as well
> though wonder if this will confuse us at some point down the road...
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/dimagi/commcare-hq-deploy/pull/115/files/592300493856e924f4ee235b699a8f6e5c088aab#r78684818>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AA4D8mEhDBAVhN6C7_3TxYg9UwnJMo1xks5qp3iCgaJpZM4J8I5F>
> .
>